### ISSUE TITLE
Markdown Guide Integration onto Scoreboard Website

### ISSUE THREAD

**Issue by waymon-ho** [KC7-Foundation/kc7_scoreboard#26]
Right now game generation process from https://github.com/KC7-Foundation/kc7 creates a custom markdown guide that takes data from the generated game and creates a mini KQL guide based on that data. The guide mirrors ones manually created in the past: https://github.com/KC7-Foundation/kc7_data/blob/main/BalloonsOverIowa/KC7%20Training%20Guide%20-%20Balloons%20Over%20Iowa.pdf

Since the guide generates a .md file, and the scoreboard webserver runs Flask, we should work to integrate the guide directly for each game. 

The scoreboard should: 
- Have a route for a "guide" page for each game that gets created via flask 
- Have the game's guide page on the nav bar on the left, once a user joins a game so they can navigate to the guide and work on it.

![image](https://user-images.githubusercontent.com/129029167/235017741-fdc011c0-dd43-4b6b-b31c-f11db1bbcc85.png)

- Use flask markdown to pull a .md file from a local location on the webserver (or if we can upload via admin panel before we host the game, if possible) and display that on the page, & render some images, formats, etc. 
- Key focus would be to figure out which stock images / templates we can re-use each guide, and which ones are required for the guide to work (i.e. it need the custom logo, etc.) from https://github.com/KC7-Foundation/kc7/blob/master/app/server/modules/constants/template_guide.txt
- The guide needs replacement for these args currently: 
**{{IMAGE_LOGO}} - Link to the Company's Logo 
{{COMPANY_DESCRIPTION}} - Long Description of Company 
{{PARTNER_DOMAIN_1_DESCRIPTION}} - Description about the Partner 
{{PARTNER_DOMAIN_2_DESCRIPTION}} - Description about the Partner
{{PARTNER_DOMAIN_3_DESCRIPTION}} - Description about the Partner
{{PARTNER_DOMAIN_4_DESCRIPTION}} - Description about the Partner** 
- The template is also re-using old pic links from hosted google content, and may need revision. 
- Maybe we should tune the guide a little bit more to standardize / not rely on pictures if it ends up being complicated? 

**ADMIN PANEL NICE TO HAVES (not mandatory)** 
- Ability to upload the .md file / required images for the game so it will load on that page
- Ability to add the missing parts of the .md, that's generated from https://github.com/KC7-Foundation/kc7/blob/master/app/server/modules/constants/template_guide.txt
- basically and find & replace func for the args listed above ^

**Comment by Boorclark** [comment]
@kkneomis How far did you get on this issue? Can we close this one or are you still working on it?

**Comment by kkneomis** [comment]
@Boorclark I'm mostly done, I want to make one final aesthetic change here
